nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,235 @@
+# Git as a Data Source
+
+The Nautobot Git as a Data Source feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot.  While using the benefits that Github already provides with issue tracking, discussions, pipelines, and approvals.
+
+For more in depth details on how this feature [see](https://nautobot.readthedocs.io/en/latest/models/extras/gitrepository/).
```

Comment:
Links like "see" or "here" are bad for SEO and should be avoided. Also, you accidentally a word.

Please change this link to be descriptive, like
```suggestion
For more technical details on how to use this feature, please see the documentation on [Git Repositories](https://nautobot.readthedocs.io/en/latest/models/extras/gitrepository/).
```